README.md

```diff
@@ -86,6 +86,9 @@ jobs:
             registry-password: ${{ secrets.REGISTRY_PASSWORD }}
             name: contoso-container
             location: 'west us'
+            ports:
+              80

```

Comment:
https://github.com/Azure/aci-deploy/blob/master/action.yml#L96
We can have them separated by space instead of a new line.

Reply:
@nosvalds can you please update this?

Reply:
Yes. I've made the fix on my Fork but for some reason this PR thinks my Fork is deleted:

https://github.com/IATI/aci-deploy/tree/port-documentation

<img width="300" alt="image" src="https://user-images.githubusercontent.com/60047271/126646820-f4f7bdaf-cbcc-4085-8c1c-39face6dec8c.png">

I might need to make a new PR...

Reply:
ohh, that's strange
So @nosvalds, planning to create a new PR then?